packages/patternfly-4/react-core/README.md

```diff
@@ -64,23 +64,25 @@ Note: All commands below assume you are on the root directory in this repository
 
 ### Install
 
-Run to install all the dependencies
+Run to install all the dependencies.
 
 ```sh
 yarn install && yarn build
 ```
 
 ### Running
 
-To start the site locally.
+To start the site locally and view the site:
 
 ```sh
 yarn start:pf4
 ```
 
+Go to [localhost:8000](http://localhost:8000/).
```

Comment:
Using a static link here could pose a couple of issues. The first is that this will show as a broken link for any crawler-bots since `http://localhost:8000/` doesn't point to anything public. Another potential trip up could be that users could be already running another service on port 8000 locally, in which case following the link would take them somewhere else. Maybe we could use a plaintext version of the address instead?